cmd/tk/create_source.go

```diff
@@ -53,6 +68,7 @@ var (
 func init() {
```

Comment:
Should have noticed this in an early review, but would it make sense to rename this to `create_gitrepository.go` and make the default `create source` type a `GitRepository`?

Reply:
The create source will have a `--helm-url` and then it will create Helm repos. Another option would be to introduce a `--type=git|helm`.